app/assets/javascripts/course/assessment/submission/answer/programming.js

```diff
@@ -1,9 +1,12 @@
 //= require helpers/form_helpers
+//= require helpers/course_helpers
 //= require helpers/answer_helpers
 //= require templates/course/assessment/submission/answer/programming/add_annotation_button
 //= require templates/course/assessment/submission/answer/programming/annotation_form
 
-(function($, FORM_HELPERS, ANSWER_HELPERS) {
+(function($, FORM_HELPERS,
```

Comment:
This way is actually pain and not extensible,  We can consider using https://larsjung.de/modulejs/ to handle modules in rails for us.